add layout tkl_nofrow_ansi_tsangan and tkl_nofrow_iso_tsangan Community Layouts

LXF-YZP opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description
add layout tkl_nofrow_ansi_tsangan and tkl_nofrow_iso_tsangan Community Layouts
* tkl_nofrow_ansi_tsangan
* tkl_nofrow_iso_tsangan

<!--- Describe your changes in detail here. -->

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

dunk2k commented:
Could this please be tested locally (and corrections applied) with a keyboard that has matching layout(s)?
Candidate keyboards include:
- cannonkeys/crin
- dnworks/frltkl
- dztech/pluto

LXF-YZP commented:
> Could this please be tested locally (and corrections applied) with a keyboard that has matching layout(s)? Candidate keyboards include:
> 
> * cannonkeys/crin
> * dnworks/frltkl
> * dztech/pluto

The two keyboard layouts I submitted are included in the following keyboard layouts.

* cannonkeys/crin
* dnworks/frltkl
* dztech/pluto

The layout provided in the keyboard in the above column has split backspace keys and split left shift and split right shift keys.

Do I need to resubmit the layout for Split backspace and split shift keys?

LXF-YZP commented:
> Could this please be tested locally (and corrections applied) with a keyboard that has matching layout(s)? Candidate keyboards include:
> 
> * cannonkeys/crin
> * dnworks/frltkl
> * dztech/pluto

What is your test keyboard layout like? Do I locally add my supplied layout to your supplied keyboard for local compilation? Thank you.

dunk2k commented:
> > Could this please be tested locally (and corrections applied) with a keyboard that has matching layout(s)? Candidate keyboards include:
> > 
> > * cannonkeys/crin
> > * dnworks/frltkl
> > * dztech/pluto
> 
> The two keyboard layouts I submitted are included in the following keyboard layouts.
> 
> * cannonkeys/crin
> * dnworks/frltkl
> * dztech/pluto
> 
> The layout provided in the keyboard in the above column has split backspace keys and split left shift and split right shift keys.
> 
> Do I need to resubmit the layout for Split backspace and split shift keys?

To maybe be more direct, I've tested locally (i.e. on my machine) these community layouts against all keyboards  with same layout (`qmk find -f contains'(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)'`, `qmk find -f contains'(layouts, LAYOUT_tkl_nofrow_iso_tsangan)'`) and same error occurs for every build.
Though the "I've tested changes" tick box is ticked, testing proves the contrary.

Could you test locally changes in this PR please, and submit (`git push`) corrections?

dunk2k commented:
> What is your test keyboard layout like? Do I locally add my supplied layout to your supplied keyboard for local compilation? Thank you.

To maybe be more direct, I've tested locally (i.e. on my machine) these community layouts against all keyboards  with same layout (`qmk find -f contains'(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)'`, `qmk find -f contains'(layouts, LAYOUT_tkl_nofrow_iso_tsangan)'`) and same error occurs for every build.
Though the "I've tested changes" tick box is ticked, testing proves the contrary.

Could you test locally changes in this PR please, and submit (`git push`) corrections?

LXF-YZP commented:
> > What is your test keyboard layout like? Do I locally add my supplied layout to your supplied keyboard for local compilation? Thank you.
> 
> To maybe be more direct, I've tested locally (i.e. on my machine) these community layouts against all keyboards with same layout (`qmk find -f contains'(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)'`, `qmk find -f contains'(layouts, LAYOUT_tkl_nofrow_iso_tsangan)'`) and same error occurs for every build. Though the "I've tested changes" tick box is ticked, local testing unfortunately proves the contrary.
> 
> Could you test locally changes in this PR please, and submit (`git push`) corrections?

I've tested locally (i.e. on my machine) these community layouts (qmk find -f contains'(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)'
The result is as follows
`
Ψ Retrieving list of keyboards with keymap "default"...
Ψ Parsing data for all matching keyboard/keymap combinations...
Ψ Filtering on condition: contains(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)...
Ψ Preparing target list...
bioi/s65:default
cannonkeys/crin:default
dnworks/frltkl:default
dztech/pluto:default
mokey/xox70hot:default
wilba_tech/wt70_jb:default
wolf/silhouette:default
`
Do I need to git push again after matching the keyboard layout above？
Maybe I still don't understand what you mean, but I appreciate your help.

dunk2k commented:
>  The result is as follows `Ψ Retrieving list of keyboards with keymap "default"... Ψ Parsing data for all matching keyboard/keymap combinations... Ψ Filtering on condition: contains(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)... Ψ Preparing target list... bioi/s65:default cannonkeys/crin:default dnworks/frltkl:default dztech/pluto:default mokey/xox70hot:default wilba_tech/wt70_jb:default wolf/silhouette:default` Do I need to git push again after matching the keyboard layout above？ Maybe I still don't understand what you mean, but I appreciate your help.

To confirm: have you tested, on your machine, these proposed Community Layouts againest keyboard(s) with matching layout(s)?

LXF-YZP commented:
> > The result is as follows `Ψ Retrieving list of keyboards with keymap "default"... Ψ Parsing data for all matching keyboard/keymap combinations... Ψ Filtering on condition: contains(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)... Ψ Preparing target list... bioi/s65:default cannonkeys/crin:default dnworks/frltkl:default dztech/pluto:default mokey/xox70hot:default wilba_tech/wt70_jb:default wolf/silhouette:default` Do I need to git push again after matching the keyboard layout above？ Maybe I still don't understand what you mean, but I appreciate your help.
> 
> To confirm: have you tested, on your machine, these proposed Community Layouts against keyboard(s) with matching layout(s)?



> > The result is as follows `Ψ Retrieving list of keyboards with keymap "default"... Ψ Parsing data for all matching keyboard/keymap combinations... Ψ Filtering on condition: contains(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)... Ψ Preparing target list... bioi/s65:default cannonkeys/crin:default dnworks/frltkl:default dztech/pluto:default mokey/xox70hot:default wilba_tech/wt70_jb:default wolf/silhouette:default` Do I need to git push again after matching the keyboard layout above？ Maybe I still don't understand what you mean, but I appreciate your help.
> 
> To confirm: have you tested, on your machine, these proposed Community Layouts against keyboard(s) with matching layout(s)?

Yes, I carried out the orders you gave me

LXF-YZP commented:
> > The result is as follows `Ψ Retrieving list of keyboards with keymap "default"... Ψ Parsing data for all matching keyboard/keymap combinations... Ψ Filtering on condition: contains(layouts, LAYOUT_tkl_nofrow_ansi_tsangan)... Ψ Preparing target list... bioi/s65:default cannonkeys/crin:default dnworks/frltkl:default dztech/pluto:default mokey/xox70hot:default wilba_tech/wt70_jb:default wolf/silhouette:default` Do I need to git push again after matching the keyboard layout above？ Maybe I still don't understand what you mean, but I appreciate your help.
> 
> To confirm: have you tested, on your machine, these proposed Community Layouts against keyboard(s) with matching layout(s)?

I'm sorry, I think I know what you mean by local execution， I'll run it locally

dunk2k commented on layouts/default/tkl_nofrow_ansi_tsangan/default_tkl_nofrow_ansi_tsangan/keymap.c:
```diff
@@ -0,0 +1,27 @@
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤     ┌───┐
+     * │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │    Shift │     │ ↑ │
+     * ├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤ ┌───┼───┼───┐
+     * │Ctrl │GUI│Alt  │                           │  Alt│App│ Ctrl│ │ ← │ ↓ │ → │
+     * └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_tkl_nofrow_ansi_tsangan(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,    KC_DEL,  KC_END,  KC_PGDN,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,             KC_UP,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL,    KC_LEFT, KC_DOWN, KC_RGHT
```
```suggestion
     * │Ctrl │GUI│Alt  │                           │  Alt│GUI│ Ctrl│ │ ← │ ↓ │ → │
     * └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘ └───┴───┴───┘
     */
    [0] = LAYOUT_tkl_nofrow_ansi_tsangan(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,    KC_DEL,  KC_END,  KC_PGDN,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,             KC_UP,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_RCTL,    KC_LEFT, KC_DOWN, KC_RGHT
```

LXF-YZP replied:
Thanks for your help, I have revised it

dunk2k commented on layouts/default/tkl_nofrow_iso_tsangan/default_tkl_nofrow_iso_tsangan/keymap.c:
```diff
@@ -0,0 +1,27 @@
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │     │ │Del│End│PgD│
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐ Ent│ └───┴───┴───┘
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ \ │    │
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤     ┌───┐
+     * │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │    Shift │     │ ↑ │
+     * ├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤ ┌───┼───┼───┐
+     * │Ctrl │GUI│Alt  │                           │  Alt│App│ Ctrl│ │ ← │ ↓ │ → │
+     * └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_tkl_nofrow_iso_tsangan(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,             KC_DEL,  KC_END,  KC_PGDN,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,             KC_UP,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL,    KC_LEFT, KC_DOWN, KC_RGHT
```
```suggestion
     * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤     ┌───┐
     * │Sft │ \ │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │    Shift │     │ ↑ │
     * ├────┴┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤ ┌───┼───┼───┐
     * │Ctrl │GUI│Alt  │                           │  Alt│GUI│ Ctrl│ │ ← │ ↓ │ → │
     * └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘ └───┴───┴───┘
     */
    [0] = LAYOUT_tkl_nofrow_iso_tsangan(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,             KC_DEL,  KC_END,  KC_PGDN,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,             KC_UP,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_RCTL,    KC_LEFT, KC_DOWN, KC_RGHT
```

dunk2k commented on layouts/default/tkl_nofrow_iso_tsangan/info.json:
```diff
@@ -0,0 +1,90 @@
+                {"x":10.5, "y":1},
+                {"x":11.5, "y":1},
+                {"x":12.5, "y":1},
+                {"x":13.5, "y":1, "w":1.5},
```
```suggestion
```

dunk2k commented on layouts/default/tkl_nofrow_iso_tsangan/info.json:
```diff
@@ -0,0 +1,90 @@
+                {"x":12.75, "y":2},
+                {"x":13.75, "y":1, "w":1.25, "h":2},
+
+                {"x":0, "y":3, "w":2.25},
```
```suggestion
                {"x":0, "y":3, "w":1.25},
                {"x":1.25, "y":3},
```

dunk2k commented on layouts/default/tkl_nofrow_iso_tsangan/layout.json:
```diff
@@ -0,0 +1,5 @@
+[{a:7},"","","","","","","","","","","","","",{w:2},"",{x:0.25},"","",""],
+[{w:1.5},"","","","","","","","","","","","","",{x:0.25,w:1.25,h:2,w2:1.5,h2:1,x2:-0.25},"",{x:0.25},"","",""],
+[{w:1.75},"","","","","","","","","","","","",""],
+[{w:2.25},"","","","","","","","","","","",{w:2.75},"",{x:1.25},""],
```
```suggestion
[{w:1.25},"","","","","","","","","","","","",{w:2.75},"",{x:1.25},""],
```

dunk2k reviewed (changes requested):
these corrections compile without issue

dunk2k commented on layouts/default/readme.md:
```diff
@@ -924,6 +939,21 @@ LAYOUT_tkl_nofrow_iso
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │ │   │   │   │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │ └───┴───┴───┘
+│      │   │   │   │   │   │   │   │   │   │   │   │   │    │
+├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤     ┌───┐
+│        │   │   │   │   │   │   │   │   │   │   │          │     │   │
+├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤ ┌───┼───┼───┐
```
```suggestion
├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤     ┌───┐
│    │   │   │   │   │   │   │   │   │   │   │   │          │     │   │
├────┴┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤ ┌───┼───┼───┐
```

dunk2k commented on layouts/default/tkl_nofrow_iso_tsangan/info.json:
```diff
@@ -0,0 +1,90 @@
+{
+    "keyboard_name": "Tenkeyless No F-Row ISO Layout With Tsangan Bottom Row",
```
```suggestion
    "keyboard_name": "Tenkeyless No F-Row ISO Layout with Tsangan Bottom Row",
```

dunk2k commented on layouts/default/tkl_nofrow_ansi_tsangan/info.json:
```diff
@@ -0,0 +1,89 @@
+{
+    "keyboard_name": "Tenkeyless No F-Row ANSI Layout With Tsangan Bottom Row",
```
```suggestion
    "keyboard_name": "Tenkeyless No F-Row ANSI Layout with Tsangan Bottom Row",
```

dunk2k commented on layouts/default/tkl_nofrow_iso_tsangan/default_tkl_nofrow_iso_tsangan/keymap.c:
```diff
@@ -0,0 +1,27 @@
+     * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤ ├───┼───┼───┤
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │     │ │Del│End│PgD│
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐ Ent│ └───┴───┴───┘
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ \ │    │
+     * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤     ┌───┐
+     * │Sft │ \ │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │    Shift │     │ ↑ │
+     * ├────┴┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤ ┌───┼───┼───┐
+     * │Ctrl │GUI│Alt  │                           │  Alt│GUI│ Ctrl│ │ ← │ ↓ │ → │
+     * └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_tkl_nofrow_iso_tsangan(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,             KC_DEL,  KC_END,  KC_PGDN,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUBS, KC_ENT,
```
```suggestion
     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ # │    │
     * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤     ┌───┐
     * │Sft │ \ │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │    Shift │     │ ↑ │
     * ├────┴┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤ ┌───┼───┼───┐
     * │Ctrl │GUI│Alt  │                           │  Alt│GUI│ Ctrl│ │ ← │ ↓ │ → │
     * └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘ └───┴───┴───┘
     */
    [0] = LAYOUT_tkl_nofrow_iso_tsangan(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,             KC_DEL,  KC_END,  KC_PGDN,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,
```

LXF-YZP replied:
Thanks for your suggestion，Would you like to know if these layouts will eventually be merged into the master branch? Is it useful to the community?

LXF-YZP replied:
I'm not sure what I'm doing makes sense
